jitpack.yml

```diff
@@ -0,0 +1,3 @@
+before_install:
+  - sdk install java 11.0.10-open
```

Comment:
Can we use Java 15 here?

Reply:
I can check, but it's about using java environment for the gradle build, not for the library artifact itself (the target jar). Java 11 is requirement for maven publishing plugin (default on jitpack is btw java 8)
The target for jvm can be set here (it is java 8 now) https://github.com/vsirotin/si-units/blob/main/kotunil/build.gradle.kts#L8
I have no particular opinion on special jvm target for now.